resources/assets/lib/user-login-observer.ts

```diff
@@ -15,7 +16,14 @@ export default class UserLoginObserver {
     dispatch(new UserLoginAction());
   };
 
-  userLogout = () => {
```

Comment:
`isNotificationEventLogoutJson` notification event handler in SocketWorker needs an update as well